tests/smoke/hello_test.go

```diff
@@ -20,6 +21,8 @@ func TestHelloBundle(t *testing.T) {
 	defer test.Close()
 	require.NoError(t, err, "test setup failed")
 
+	// Set up a fresh docker environment
+	mgx.Must(shx.RunV("docker", "system", "prune", "--all", "-f"))
```

Comment:
This is a very expensive call to make on a dev machine, since for example I have 30-60GB of cached images that I would have to repull after every time I run this test. 😱  Plus this is also removing containers and volumes too which is not okay to just delete on people's machines.

A less invasive way to make sure that we don't have that image in our cache is to use the `docker image rm IMAGE` command, without any flags specified so that if it's in use, the dev gets an error and can figure out what is a safe way to cleanup their environment.